public/js/components/Editor.js

```diff
@@ -221,22 +223,30 @@ const Editor = React.createClass({
     }
   },
 
-  render() {
+  renderBreakpoints() {
     const { breakpoints, sourceText } = this.props;
     const isLoading = sourceText && sourceText.get("loading");
 
+    if (isLoading) {
+      return;
+    }
+
+    return breakpoints.valueSeq().map(bp => {
+      return Breakpoint({
+        key: makeLocationId(bp.location),
+        breakpoint: bp,
+        editor: this.editor && this.editor.codeMirror
+      });
+    });
+  },
+
+  render() {
     return (
       dom.div(
         { className: "editor-wrapper devtools-monospace" },
         dom.div({ className: "editor-mount" }),
-        !isLoading &&
-          breakpoints.valueSeq().map(bp => {
-            return Breakpoint({
-              key: makeLocationId(bp.location),
-              breakpoint: bp,
-              editor: this.editor && this.editor.codeMirror
-            });
-          })
+        this.renderBreakpoints(),
```

Comment:
I'm not sure this render function is big enough to be split up. It's nice to have things in one place when possible, and only split it up when it gets too large. The `renderBreakpoints` function is several lines longer than this expression :)